advanced/am-pattern-2/templates/am/wso2am-pattern-2-conf-entrypoint.yaml

```diff
@@ -20,62 +20,62 @@ metadata:
 data:
```

Comment:
@Chathuru a suggestion to improve clarity. This file is for APIM pattern 2. But, this PR does not cover APIM pattern 2 as of now. Shall we integrate this change to the other PR in which you plan on adding APIM pattern 2 changes?

Reply:
Fin in commit https://github.com/wso2/kubernetes-apim/pull/346/commits/5b63640f3624a2a1bcf23b3f92523488b87e5aa5